instrumentserver/client/proxy.py

```diff
@@ -433,6 +434,41 @@ def paramsFromFile(self, filePath: str):
         else:
             logger.warning(f"File {filePath} does not exist. No params loaded.")
 
+class SubClient(QObject):
+    """Test client to test PUB-SUB"""
+    #: Signal(str) --
+    #: emitted when the server broadcast either a new parameter or an update to an existing one
+    update = pyqtSignal(str)
+
+    def __init__(self, host='localhost', port=5556):
+        super(QObject, self).__init__()
+        self.host = host
+        self.port = port
+        self.addr = f"tcp://{host}:{port}"
+
+
+    def connect(self):
+        logger.info(f"Connecting to {self.addr}")
+        context = zmq.Context()
+        socket = context.socket(zmq.SUB)
+        socket.connect(self.addr)
+
+        # subscribe to changes in parameters and creating of parameters.
+        socket.setsockopt_string(zmq.SUBSCRIBE, 'set')
```

Comment:
the category names might become unclear in  the future. 
i'd suggest something like ``parameter-update`` and ``parameter-creation``